rmf_building_map_tools/building_map/level.py

```diff
@@ -174,17 +174,18 @@ def generate_walls(self, model_ele, model_name, model_path):
                 model_path,
                 self.transformed_vertices)
 
-    def generate_sdf_models(self, world_ele):
+    def generate_sdf_models(self, world_ele, dae_export_plugin):
         for model in self.models:
             model.generate(
                 world_ele,
                 self.transform,
                 self.elevation)
 
         # sniff around in our vertices and spawn robots if requested
-        for vertex_idx, vertex in enumerate(self.vertices):
-            if 'spawn_robot_type' in vertex.params:
-                self.generate_robot_at_vertex_idx(vertex_idx, world_ele)
+        if dae_export_plugin is False:
+            for vertex_idx, vertex in enumerate(self.vertices):
+                if 'spawn_robot_type' in vertex.params:
+                    self.generate_robot_at_vertex_idx(vertex_idx, world_ele)
```

Comment:
I wonder if we should limit the filtered objects to only robots or to all the dynamic objects, i.e. lifts / doors.
What would you recommend?

Reply:
yup i forgot about that, we should remove those

Reply:
removed ac3081a